common/Makefile.am.inc

```diff
@@ -142,9 +142,10 @@ libflatpak_common_la_CFLAGS = \
 	$(SYSTEMD_CFLAGS) \
 	$(DCONF_CFLAGS) \
 	-DLIBEXECDIR=\"$(libexecdir)\" \
+	$(EOS_PARENTAL_CONTROLS_CFLAGS) \
 	-I$(srcdir)/dbus-proxy \
 	$(NULL)
-libflatpak_common_la_LIBADD = $(AM_LIBADD) libglnx.la $(BASE_LIBS) $(OSTREE_LIBS) $(SOUP_LIBS) $(JSON_LIBS) $(XAUTH_LIBS) $(LIBSECCOMP_LIBS) $(INTERNAL_GPGME_LIBS) $(SYSTEMD_LIBS) $(DCONF_LIBS)
+libflatpak_common_la_LIBADD = $(AM_LIBADD) libglnx.la $(APPSTREAM_GLIB_LIBS) $(BASE_LIBS) $(OSTREE_LIBS) $(SOUP_LIBS) $(JSON_LIBS) $(XAUTH_LIBS) $(LIBSECCOMP_LIBS) $(INTERNAL_GPGME_LIBS) $(SYSTEMD_LIBS) $(DCONF_LIBS) $(EOS_PARENTAL_CONTROLS_LIBS)
```

Comment:
This is my fault because it was introduced in the last rebase, but this commit should only be adding `EOS_PARENTAL_CONTROLS_LIBS` not `APPSTREAM_GLIB_LIBS` and the latter should be added by the commit "Revert "Stop linking libflatpak against listappstream-glib""

Reply:
Rectified in the latest iteration and mentioned the rectification in relevant commit's message.